src/components/Nav/index.tsx

```diff
@@ -92,16 +104,44 @@ const Nav: FC<IProps> = ({ path }) => {
                   variant="ghost"
                   isSecondary
                   px={1.5}
+                  _hover={{
+                    transform: "rotate(30deg)", // Rotate the icon to 30 degrees on hover
+                    color: "blue", // Change color to blue on hover
+                  }}
                   onClick={toggleColorMode}
                 ></IconButton>
+
                 <ButtonLink
+                  onMouseOver={() => setLanguagesHover(true)}
+                  onMouseOut={() => setLanguagesHover(false)}
+                  as={Link}
                   to={`/languages/${fromPageParameter}`}
-                  leftIcon={<Icon as={MdLanguage} />}
                   variant="ghost"
                   isSecondary
                   px={1.5}
+                  _hover={{
+                    color: "blue", // Change color to blue on hover
+                  }}
+                  style={{
+                    display: "inline-flex",
+                    alignItems: "center",
+                  }}
```

Comment:
This is excessive, as the Chakra `Button` being rendered already supplies these styles.
```suggestion
```